### PR TITLE
Fixed some pylint issues in module tcms.testruns.models

### PR DESCRIPTION
**Pull request by asankov**
Small fixes for some of the issues in `tcms.testruns.models`

### CODE REVIEW

File: tcms/report/views.py [hunk 1]
```diff
@@ -442,10 +442,10 @@ def _report_data_context(self):
                 self._data.generate_status_matrix(build_ids)
             )
 
-            status_ids = (TestCaseRunStatus.id_failed(),)
+            status_ids = (TestCaseRunStatus.with_status_id(TestCaseRunStatus.FAILED),)
             failed_case_runs = self.read_case_runs(build_ids, status_ids)
 
-            status_ids = (TestCaseRunStatus.id_blocked(),)
+            status_ids = (TestCaseRunStatus.with_status_id(TestCaseRunStatus.BLOCKED),)
```
Comment (atodorov) [inline tcms/report/views.py]: You can also use `._status_to_id()` directly. Maybe update the name to not start with `_`. We don't need extra layers of methods wrapping other methods, wrapping objects.get

File: tcms/testruns/models.py [hunk 2]
```diff
@@ -309,6 +311,10 @@ def stats_caseruns_status(self, case_run_statuses=None):
 
 
 class TestCaseRunStatus(TCMSActionModel):
+    FAILED = 'failed'
+    BLOCKED = 'blocked'
+    PASSED = 'passed'
```
Comment (atodorov) [inline tcms/testruns/models.py]: These are status names and are always upper case.

Comment (asankov) [reply tcms/testruns/models.py]: `@classmethod def id_failed(cls): return cls._status_to_id('failed')` This is the implementation of the method I just deleted. I took the status string from there. Will change it now, but has it not been working propertly before?

File: tcms/report/views.py [hunk 3]
```diff
@@ -442,10 +442,10 @@ def _report_data_context(self):
                 self._data.generate_status_matrix(build_ids)
             )
 
-            status_ids = (TestCaseRunStatus.id_failed(),)
+            status_ids = (TestCaseRunStatus.status_to_id(TestCaseRunStatus.FAILED),)
```
Comment (atodorov) [inline tcms/report/views.py]: I think it's best to directly use `objects.get()` with the class attribute constants (w/o the extra upcase()) and without the extra try-except coming from `status_to_id()`. This way we will get failures when there are no such objects so they can be fixed faster.

Note: searching by name can fail when the names are translated at the DB level.

**Comment by asankov** [comment]
I have applied the comments, and also removed some other unused methods from this class

**Pull request merged**